workflow/usage.rst

```diff
@@ -1,477 +0,0 @@
-.. index::
```

Comment:
by removing this file, can you just see this one ?
https://github.com/symfony/symfony-docs/pull/10751/files
do you think we can merge both PR together?